docs/docs/features/index.mdx

```diff
@@ -32,6 +32,16 @@ subset of users (like beta users), gradually roll out a new features, or turn an
 4. Add [Override Rules](/features/rules) to target users with a Forced Value, rollout your feature to a percentage of users, or run an experiment
 5. Learn about [Scheduling](/features/scheduling) features and different feature [Environments](/features/environments)
 
+### Requiring Approvals on Feature changes
+
+You are able to require draft feature flags to undergo a review before publishing.
+The only exception to this rule is if you are an admin, you can override the review process from the request review modal.
+
+To set your organization to require reviews:
+1. Go to [General Settings](/settings) in GrowthBook.
```

Comment:
This markdown will render on `docs.growthbook.io`, so relative links don't work.  In this case, I'd probably just not make this a link at all.  The URL will be different depending on if someone is self-hosting or using Cloud